lib/hawkular/version.rb

```diff
@@ -4,5 +4,5 @@
 # @see https://github.com/hawkular
 module Hawkular
   # Version of the Hawkular Ruby Gem
-  VERSION = '2.2.1'
+  VERSION = '2.3.0'.freeze
```

Comment:
never saw a freeze on a version - it shouldn't matter in this case imo, but I guess it's harmless